### ISSUE TITLE
length - minimizeAutoencoder

### ISSUE THREAD

**Issue by j0ntt** [saviola777/darch2#1]
What is the correct form of "length" parameter in function minimizeAutoencoder?

I have been testing it with e.g. 1,10,-5,c(2,3) but nothing is working. I get this error message:

Error in data * log(output) : non-conformable arrays

backpropagation and rpropagation work just fine.

**Comment by saviola777** [comment]
I haven't looked into Conjugate Gradients fine-tuning at all since starting to work on darch, as it looked like a major headache that I wanted to deal with later.

That said, I'll try testing / debugging it on the weekend and fix it if it's a small error. If it's a bigger issue it might take longer, but I'll report back here in any case.

Thanks for reporting the issue.
